tests/regressions/execution_tree/random_d_1139.cpp

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2020 Bita Hasheminezhad
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#include <phylanx/phylanx.hpp>
+
+#include <hpx/hpx_init.hpp>
+#include <hpx/include/iostreams.hpp>
+#include <hpx/include/lcos.hpp>
+#include <hpx/include/util.hpp>
+#include <hpx/testing.hpp>
+
+#include <cstdint>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include<iostream>
+
+///////////////////////////////////////////////////////////////////////////////
+void execution_time(std::string const& name, std::string const& codestr)
+{
+    phylanx::execution_tree::compiler::function_list snippets;
+    phylanx::execution_tree::compiler::environment env =
+        phylanx::execution_tree::compiler::default_environment();
+
+    auto const& compiled_code =
+        phylanx::execution_tree::compile(name, codestr, snippets, env);
+
+    hpx::cout << "Locality: " << hpx::get_locality_id()
+              << "  Starting execution for: " << name << hpx::endl;
+    std::uint64_t t = hpx::util::high_resolution_clock::now();
+    phylanx::execution_tree::primitive_argument_type a = compiled_code.run().arg_;
+    t = hpx::util::high_resolution_clock::now() - t;
+
+    std::cout << *(a.annotation()) << "\n\n\n";
```

Comment:
I think this std::cout was good for demonstration, but we don't need it anymore and we can remove #include<iostream> too.

Reply:
@taless474 right. Please feel free to push to this branch to apply these changes.